apps/extension/next-auth.d.ts

```diff
@@ -1,12 +0,0 @@
-import { DefaultSession } from "next-auth"
```

Comment:
having to add next-auth as dep and duplicate the module augementation isn't very optimal and good dx, so i included the type through the tsconfig isntead